database/migrations/2019_11_12_171528_songs.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class Songs extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('songs', function (Blueprint $table) {
+            $table->increments('id');
+            $table->unsignedInteger('user_id');
+            $table->string('title');
+            $table->json('songText');
+            $table->string('category');
+            $table->string('artist');
+            $table->string('slug');
+            $table->boolean('is_accepted');
+            $table->foreign('user_id')
+                ->references('id')
+                ->on('users')
+                ->onDelete('cascade');
```

Comment:
It's probably a bad idea. Do we really want to delete all user songs when the user account gets deleted?